private/bufpkg/bufimage/build_image.go

```diff
@@ -80,14 +82,15 @@ func buildImage(
 	if buildResult.Err != nil {
 		return nil, buildResult.Err
 	}
-	fileDescriptors, err := checkAndSortFileDescriptors(buildResult.FileDescriptors, paths)
+	sortedFiles, err := checkAndSortFiles(buildResult.Files, paths)
```

Comment:
I tweaked the names of a few things in here for better consistency and readability. In an earlier PR, I had changes some of the types from `protodescriptor.FileDescriptor` to `linker.File`, but didn't change the corresponding "FileDescriptor" names. This fixes that omission.